modules/arrow/src/triangulate-on-worker.ts

```diff
@@ -21,17 +22,24 @@ export type TriangulationWorkerOutput =
 
 export type ParseGeoArrowInput = {
   operation: 'parse-geoarrow';
-  arrowData: ArrayBuffer;
+  chunkData: {
```

Comment:
Interesting that you choose to use the Arrow API on the worker. 

A problem with Arrow is that tables as JS classes (data + methods) and they can't be transferred to workers.

To call this worker we need to extract the binary arrays from arrow. 

What do we gain by putting them back into arrow again to do our calculations?

Reply:
My idea here is to pass the pure data (not the methods in the class) to the web worker, so we can reconstruct the arrow.Data and then arrow.Vector for parsing the specific chunk. Most of the pure data are meta data like offset, indices, length etc.